### ISSUE TITLE
Invalid escape sequences in tests

### ISSUE THREAD

**Issue by felixschurk** [GothenburgBitFactory/taskwarrior#3388]
When running the python tests, warnings about invalid escape sequences, as in #3316 do appear all over the place. 

```sh
SyntaxWarning: invalid escape sequence
```

In order to fix them, one can prefix the corresponding string with an `r` to have it as a raw string.
Probably this corresponds with the used Python version, which is currently under Fedora 40 `Python 3.12.2`.

**Comment by djmitche** [comment]
Huh, did we just fix a few of these in #3322 and miss the rest?

**Comment by felixschurk** [comment]
I am pretty sure that it corresponds to the change to out-of-source build #3271, I am really sorry for missing that part.
So as only the ones in `problems` was reported I only fixed them there.

Now while working with the test I realized (especially after also seeing the invalid escape) that unfortunately we dropped all the integration python tests with this change. (Due to not updated, copied python tests as in #3387 and the corresponding issue. 

When one does look up the commit history and the corresponding unit test one can see that [here](https://github.com/GothenburgBitFactory/taskwarrior/actions/runs/7852245744/job/21430388252) around 3800 tests where run and then in [this run](https://github.com/GothenburgBitFactory/taskwarrior/actions/runs/8039644577/job/21956856198) only 2700.
